ResourceStatusSystem/scripts/dirac-rss-sync.py

```diff
@@ -149,17 +149,18 @@ def initSites():
     DIRACExit( 1 )
 
   for site, elements in sites['Value'].iteritems():
+    elementType = site.split( '.' )[0]
     parameters = { 'status': elements[0],
                    'reason': 'Synchronized',
                    'name': site,
                    'dateEffective': elements[1],
                    'tokenExpiration': Datetime,
-                   'elementType': 'Site',
+                   'elementType': elementType,
                    'statusType': 'all',
                    'lastCheckTime': None,
                    'tokenOwner': elements[2] }
 
-    result = rssClient.addIfNotThereStatusElement( "Site", "Status", **parameters )
+    result = rssClient.updateStatusElement( "Site", "Status", **parameters )
```

Comment:
The problem is that if addIfNotThereStatusElement is used, the parameters are not updated. By this point ALL the sites must be in the table, they were just added before. In reality, we can have in the Site Mask sites that are no more in CS. Those sites should not be added at this point.